### ISSUE TITLE
Clarify Label of "Copy All Content"

### ISSUE THREAD

**Issue by mrwweb** [WordPress/gutenberg#11739]
**Describe the bug**
The "Copy All Content" action copies the HTML source with comments onto the clipboard. This did not match my expected behavior.

This ability to copy rich text allows for pasting into emails, Word, Google Docs, etc. and also requires that the user paste into the Visual tab of the Classic Editor.

Related: #9018

**Expected behavior**
I didn't expect "Content" to mean "HTML Code." If the behavior isn't going to change, this could be most easily resolved by clarifying the label of the button to something like "Copy Post as HTML".

I also _wanted_ to use this button for pasting content into the Visual Mode of the Classic Editor, so a second button of "Copy Post Content" for this purpose (and Word, etc.) would be valuable. Alternately, the button could be "Copy Post Content..." and then lead to a modal/something with options:

1. Copy Post as HTML
1. Copy Post as HTML without comments
1. Copy Rich Text Post Content

**Desktop (please complete the following information):**
Win 10, Firefox 63, WP 5.0-beta3, no plugins, Twenty Nineteen

**Comment by mrwweb** [comment]
Sitting at WordCamp Seattle allowed for some great quick user testing.

I showed two people the following:

1. Click "Copy All Content"
2. Open Google Docs
3. Ask what they expect to happen when I past
4. Paste

Both people expected the content to appear as it did in the Visual Editor mode of Gutenberg and not to paste HTML which is the current behavior.

Both commented that the menu item label violated their expectations of what would happen.

**Comment by michelleweber** [comment]
Yeah, I wouldn't expect the HTML either. "Copy post as HTML" or "Copy post HTML" would be a lot clearer. Or if you want to go all-out "Copy complete post content as HTML."

**Comment by DeveloperWil** [comment]
Anyone working on a way to copy the plain-text form a WordPress block post or page, rather than the HTML?

**Comment by mrwweb** [comment]
Closing this issue as the button has been renamed "Copy all blocks" where I think the HTML-copying behavior is more understandable. If people are still wanting an option to copy a post with limited semantic formatting or in markdown, that would probably be better addressed in a new issue specifically addressing use cases for those features..
